### DISCUSSION TITLE
Check merged cuts in exported file

### DISCUSSION THREAD

**Discussion by ceekei** [mifi/lossless-cut#1886] [Support] [answered]
Hi!

Is there a function to check the cuts in the exported file? For example I have several cuts in the original file to cut out commercials. I want to jump to the time 2 seconds before end of the first segment, play the exported file from the corresponding time on to check if the cut and merge is clean. Playback maybe until second 2 of segment 2, jumping 2 seconds before end of segment 2 to check the next cut and merge point and so on. 

I recognized the function "play segments in order" but this plays the original file and the whole segment. This is not I want to do. 

Is this implemented already and I cannot find it? 

Why this? I recognize some sinlge frames with commercial stuff at some cut points. Sometimes it helps to set the cut points a few frames before or after the current. A function like I mentioned would help to check quickly if everything was merged fine.

**Comment by mifi** [comment]
Hi. I think what you're asking is similar to #1756 - I've opened a new issue #1887 to gather information about this feature

**Reply by ceekei** [reply]
Found #622, seems to be the same as I mentioned.

**Comment by ceekei** [comment]
Thanks for the fast reply. #1756 seems to me a playback of the original file. I meant a playback of the exported file. The rest seems to be the same feature request. 

I am asking playing the exported file because sometimes the cuts in the original file seems to be okay, so the playback of the original file at the cut points will be fine but not the exported one. So it is a simple 'jump to some certain points in the exported file and play for some seconds there' - Function

**Reply by mifi** [reply, marked as answer]
Ah, ok. so you want to actually:
1. define the cut segments
2. press export
3. wait for export to finish
4. click on some button that will trigger the "Loop beginning and end of current segment"-function, but on the exported file instead of original?

**Reply by ceekei** [reply]
Correct, and of course if there are more than two segments you should be able to to jump to all of the cutting points in the exported file.

**Reply by mifi** [reply]
I've updated #1887 with a proposed solution
